TYPO3.Flow/Documentation/TheDefinitiveGuide/PartIII/ObjectManagement.rst

```diff
@@ -1321,7 +1321,7 @@ and so on, the performance in production context can be improved::
 	 * @return array Array of method parameters by action name
 	 * @Flow\CompileStatic
 	 */
-	static public function getActionMethodParameters($objectManager) {
+	static protected function getActionMethodParameters($objectManager) {
```

Comment:
I think it's good measure to make this helper method protected in general, so I adjusted the example here